src/coreclr/vm/gchelpers.cpp

```diff
@@ -803,7 +803,7 @@ OBJECTREF   DupArrayForCloning(BASEARRAYREF pRef)
 //
 // Helper for parts of the EE which are allocating arrays
 //
-OBJECTREF AllocateObjectArray(DWORD cElements, TypeHandle elementType, BOOL bAllocateInLargeHeap)
+OBJECTREF AllocateObjectArray(DWORD cElements, TypeHandle elementType, BOOL bAllocateInLargeHeap, BOOL bAllocateInPinnedHeap)
```

Comment:
Is the `bAllocateInLargeHeap` argument useful? I think the place that you have touched was the only place where it was used.